pkg/cli/cliflags/flags.go

```diff
@@ -228,12 +228,19 @@ server to listen on an external address in insecure mode.`,
 		Description: `Define the maximum number of results that will be retrieved.`,
 	}
 
+	SetPassword = FlagInfo{
+		Name:   "set-password",
+		EnvVar: "COCKROACH_SET_PASSWORD",
```

Comment:
This flag should not have an environment variable. We only want to support environment variables for flags that you might want to set the same way every time you run cockroach.